readthedocs/core/management/commands/update_repos.py

```diff
@@ -26,22 +26,30 @@ class Command(BaseCommand):
     """Management command for rebuilding documentation on projects"""
 
     help = __doc__
-    option_list = BaseCommand.option_list + (
```

Comment:
It seems that we lose the positional arguments:

    manage.py update_repos mkdocs

doesn't work anymore.

This command is used when `import_from_live` is called.

From the example at https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/#custom-commands-options we need some extra lines to allow that.

I had to add this to make it work.

            parser.add_argument('slugs', nargs='+', type=str)

Probably similar work is needed in the rest of the commands.

cc @stsewd

Reply:
I'll see if I can fix this today